src/ngraph/builder/reshape.cpp

```diff
@@ -173,3 +174,56 @@ shared_ptr<Node> builder::expand_dims(const Output<Node>& value, size_t axis)
                value, get_default_order(value.get_shape().size()), output_shape)
         ->add_provenance_group_members_above({value});
 }
+
+shared_ptr<Node> builder::opset1::reshape(const Output<Node>& value, const Shape& shape)
+{
+    const auto out_pattern = op::Constant::create(
+        element::i64, Shape{shape.size()}, vector<int64_t>(shape.begin(), shape.end()));
+    const bool special_zero = false;
+    return make_shared<op::v1::Reshape>(value, out_pattern, special_zero)
```

Comment:
can we use make_shared<opset1::Reshape>(...) instead of directly calling out the version.

Reply:
done